jaggr-core/WebContent/loaderExtCommon.js

```diff
@@ -346,10 +346,14 @@ var params = {
 				addFoldedModuleName(dep, i, oFolded, oPrefixes);
 			}
 		}
-		return url + (argNames[0] === 'modules' ? ((url.indexOf("?") === -1 ? "?" : "&") + "count=" + i) : "" ) + 
-		             (sizeofObject(oFolded) ? ("&"+argNames[0]+"="+encodeURIComponent(encodeModules(oFolded))):"") + 
-		             (ids.length ? ("&"+argNames[1]+"=" + base64EncodeModuleIds(ids, base64Encoder, argNames[0] === 'modules' && hash)):"");
+		url +=(argNames[0] === 'modules' ? ((url.indexOf("?") === -1 ? "?" : "&") + "count=" + i) : "" ) + 

```

Comment:
This looks fine logic-wise... but these nested and chained ternary expressions are not the easiest to follow...  

Do you think we can break these up to if statements to craft the URL?